corehq/apps/userreports/tasks.py

```diff
@@ -141,6 +142,7 @@ def rebuild_indicators_in_place(indicator_config_id, initiated_by=None, source=N
     with notify_someone(initiated_by, success_message=success, error_message=failure, send=True):
         adapter = get_indicator_adapter(config)
         if not id_is_static(indicator_config_id):
+            config.meta.build.awaiting = False
```

Comment:
Maybe the following suggestion can be done in a different PR... 
Since it seems here's a lot happening when the data source build changes state (as well as on line 116-119), I want to suggest that we wrap the build state transitions with appropriate methods, for example here the (re)build starts and a bunch of flags are being set - it would be nice if a method handled that such that we can simply do stuff like

```python
>>> config.set_build_queued
>>> config.set_build_started
>>> config.set_build_finished
```

This way we handle the state flags in one place.